hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/AbstractHoodieWriteClient.java

```diff
@@ -608,7 +627,7 @@ public boolean rollback(final String commitInstantTime, boolean skipLocking) thr
           .findFirst());
       if (commitInstantOpt.isPresent()) {
         LOG.info("Scheduling Rollback at instant time :" + rollbackInstantTime);
-        Option<HoodieRollbackPlan> rollbackPlanOption = table.scheduleRollback(context, rollbackInstantTime,
+        Option<HoodieRollbackPlan> rollbackPlanOption = pendingRollback ? Option.of(pendingRollbackInfo.getValue()) : table.scheduleRollback(context, rollbackInstantTime,
```

Comment:
NTR: I could not use Option.of().OrElse, bcoz, table.scheduleRollback already returns an Option<HoodieRollbackPlan> and so Option.of(pendingRollbackInfo.getValue()).orElse( table.scheduleRollback(...) ) does not work.